docs/r2/tensorboard_quickstart.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>Linkify "TensorBoard callback" to https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/TensorBoard.</p><p><br></p><p>You may also want to mention that you're turning on histogram computation every epoch. By default it's off.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/1870/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
I added a link to the r2 version of the docs. We need to figure out how all of this gets versioned appropriately.